cmd/uwsgi_exporter/main_test.go

```diff
@@ -182,7 +209,13 @@ func waitForBody(urlToGet string) (body []byte, err error) {
 
 // getBody is a helper function which retrieves http body from given address.
 func getBody(urlToGet string) ([]byte, error) {
-	resp, err := http.Get(urlToGet) //#nosec
+	client := &http.Client{}
+	ctx := context.Background()
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, urlToGet, http.NoBody)
```

Comment:
```suggestion
	req, err := http.NewRequest(http.MethodGet, urlToGet, http.NoBody)
```
If we use `context.Background()` using `http.NewRequest` is already sufficient as it uses this context internally. See https://pkg.go.dev/net/http@go1.24.3#NewRequest